samples/azure-quantum/hello-world/README.md

```diff
@@ -0,0 +1,26 @@
+---
+page_type: sample
+author: alchocro
+description: Hello World Samples
+ms.author: alchocro@microsoft.com
+ms.date: 08/08/2022
+languages:
+- python
+products:
+- azure-quantum
+---
+
+# Hello World Samples
+
+These are Azure Quantum sample notebooks that illustrate how to use Q#, Qiskit, or Cirq to write a simple program and run it against an Azure Quantum provider.
+
+## Manifest
+
+- [HW-ionq-cirq.ipynb](./HW-ionq-cirq.ipynb): Azure Quantum notebook for running a simple Cirq program on the IonQ simulator
```

Comment:
I believe these should be absolute links?

Reply:
Fixed!